pkg/kritis/review/validating_transport.go

```diff
@@ -101,51 +103,74 @@ func (avt *AttestorValidatingTransport) validatePublicKeyId(pubKey v1beta1.Publi
 	return nil
 }
 
+// NOTE: I am considering changing this function signature to:
+// func (avt *AttestorValidatingTransport) CheckValidatedAttestation(image string) error
+// The function would propagate the error (or success) of the
+// verifier.VerifyAttestation call. It currently returns ValdiatedAttestations,
+// which contain unused and not particularly meaningful information.
+// This is not a trivial change, and would be done in a follow-up PR.
 func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) ([]attestation.ValidatedAttestation, error) {
-	keys := map[string]v1beta1.PublicKey{}
+	publicKeys := []cryptolib.PublicKey{}
 	numKeys := len(avt.Attestor.Spec.PublicKeys)
-	for i, pubKey := range avt.Attestor.Spec.PublicKeys {
-		if err := avt.validatePublicKey(pubKey); err != nil {
+	validatedAtts := []attestation.ValidatedAttestation{}
+
+	for i, attestorKey := range avt.Attestor.Spec.PublicKeys {
+		if err := avt.validatePublicKey(attestorKey); err != nil {
 			// warning level because single key failure is something tolerable
 			glog.Warningf("Error parsing key %d (%d keys total) for %q: %v", i, numKeys, avt.Attestor.Name, err)
 			continue
 		}
-		if _, ok := keys[pubKey.KeyId]; ok {
-			glog.Warningf("Overwriting key with same fingerprint %s for %q.", pubKey.KeyId, avt.Attestor.Name)
+		// Base64 decode the attestor's key
```

Comment:
nit: remove this comment because it does not explain anything that is not obvious from the code

Reply:
Done.